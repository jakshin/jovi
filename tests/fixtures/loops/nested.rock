Pings is 0

while Pings is lower than 3
  say "--> PING"
  build Pings up
  Pongs is 0
  until Pongs is 3
    say "    pong"
    build Pongs up
  (end until)
(end while)
