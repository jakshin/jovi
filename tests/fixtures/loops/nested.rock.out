--> PING
    pong
    pong
    pong
--> PING
    pong
    pong
    pong
--> PING
    pong
    pong
    pong
